packages/lib-classifier/src/store/WorkflowStepStore/Step/Step.spec.js

```diff
@@ -59,6 +61,30 @@ describe('Model > Step', function () {
     })
   })
 
+  describe('with any incomplete, optional tasks', function () {
+    let tasks
+
+    const pointToolWithMin = {
+      help: '',
+      label: 'Point please.',
+      min: 1,
+      type: 'point'
+    }
+
+    before(function () {
+      tasks = [
+        MultipleChoiceTask.TaskModel.create(MultipleChoiceTaskFactory.build({ taskKey: 'T1', required: '' })),
+        SingleChoiceTask.TaskModel.create(SingleChoiceTaskFactory.build({ taskKey: 'T2', required: '' })),
+        DrawingTask.TaskModel.create(DrawingTaskFactory.build({ taskKey: 'T3', required: '', tools: [pointToolWithMin] }))
```

Comment:
Small thing (not blocking), but `task.required` defaults to `false` in the project builder.

Reply:
nice catch, thank you!